FirebaseCombineSwift/Tests/Unit/Storage/StorageReferenceTests.swift

```diff
@@ -54,16 +54,9 @@ class StorageReferenceTests: XCTestCase {
     // When
     ref?.putFile(from: dummyFileURL, metadata: nil)
       .sink { completion in
-        if case let .failure(error as NSError) = completion {
+        if case let .failure(error) = completion {
           putFileExpectation.fulfill()
-
-          XCTAssertEqual(error.domain, StorageErrorDomain)
-          XCTAssertEqual(error.code, StorageErrorCode.unknown.rawValue)
-
-          let expectedDescription =
-            "File at URL: \(dummyFileURL.absoluteString) is not reachable. Ensure file URL is not" +
-            " a directory, symbolic link, or invalid url."
-          XCTAssertEqual(error.localizedDescription, expectedDescription)
+          XCTAssertEqual("unknown", "\(String(describing: error))")
```

Comment:
nit: i think this can be simplified to
```suggestion
          XCTAssertEqual("unknown", String(describing: error))
```